getanewsletter.php

```diff
@@ -288,7 +290,7 @@ function display_subscription_forms_list($connectionSucceeded, $forms) {
     <?php
 }
 
-function display_subscription_form($attributes, $lists, $currentFormData, $form_id = null) {
+function display_subscription_form($attributes, $lists, $currentFormData, $form_id = null, $senders) {
```

Comment:
@kayart I got a warning about this line in my PHP log, see below:

`[14-Nov-2024 08:18:40 UTC] PHP Deprecated:  Optional parameter $form_id declared before required parameter $senders is implicitly treated as a required parameter in .../wordpress/wp-content/plugins/getanewsletter/getanewsletter.php on line 293`

Is this something we can fix?